website/source/docs/guides/production-acls.html.md

```diff
@@ -0,0 +1,484 @@
+rolling restart. After applying the token. 
+
+#### Advantages of the ACL HTTP API
+
+Alternatively, you can apply the agent token with the [HTTP
+API](https://www.consul.io/api/acl/tokens.html#create-a-token). The HTTP API
+has three advantages.
+
+1. The token can be applied immediately to the agent, no service restart required.
+2. The configuration file does not need to be manually updated.  The token will
+not be persisted to disk, it will not be stored in plain text. 
+3. The final advantage can also be a disadvantage. Since the token is not
+persisted to disk, any time the agent is restarted the token will need to
+re-applied. This may not be an issue for scheduled maintenance, but could
+complicate an unplanned server outage.  
+
+Now that the agents have tokens, we can
+create tokens for the services.  
+
+## Apply Individual Tokens to the Services
+
+The token creation and application process for services is similar to agents. 
+Create a policy.  Use that policy to create a token.  Add the token to the
+service configuration file. 
+
+Review the [service
+rules](https://www.consul.io/docs/agent/acl-rules.html#service-rules) before
+getting started.
+
+Below is an example service definition that needs a token after bootstrapping
+the ACL system.
+
+``` 
+{ 
+  "service": { 
+    "name": "dashboard", 
+    "port": 9002, 
+    "check": { 
+    	"id": "dashboard-check", 
+    	"http": "http://localhost:9002/health", 
+    	"method": "GET",
+    	"interval": "1s", 
+    	"timeout": "1s" 
+    	} 
+    } 
+} 
+```
+
+This service definition should be located in the [configuration
+directory](https://www.consul.io/docs/agent/options.html#_config_dir) on one of
+the clients.  
+
+First, create the policy that will grant write privileges to only the
+"dashboard" service. This means the "dashboard" service can register the
+itself, update it's health checks, and write any of the fields in the [service
+definition](https://www.consul.io/docs/agent/services.html).
+
+```
+# dashboard-policy.hcl
+service "dashboard" { 
+	policy = "write" 
+} 
+```
+
+Use the policy definition to initiate the policy.
+
+``` 
+$ consul acl policy create -name "dashboard-service" -rules @dashboard-policy.hcl 
+```
+
+Next, create a token with the policy.
+
+``` 
+$ consul acl token create -description "Token for Dashboard Service" -policy-name dashboard-service 
+```
+
+The command will return information about the token, which should include a
+description and policy information. As usual, save the token to a secure
+location.
+
+
+Finally, add the token to the service definition. 
+
+``` 
+{ 
+  "service": { 
+  	"name": "dashboard", 
+  	"port": 9002, 
+  	“token”: “57c5d69a-5f19-469b-0543-12a487eecc66”, 
+  	"check": { 
+  		"id": "dashboard-check",
+  		"http": "http://localhost:9002/health", 
+  		"method": "GET", 
+  		"interval": "1s",
+  		"timeout": "1s" 
+  		} 
+  	} 
+ } 
+```
+
+If the service is running, you will need to restart it. Unlike agent tokens,
```

Comment:
This token is just used for agent anti-entropy and for (de)registering the service and its associated checks. If the **real** service actually needs to access Consul for other reasons then it will need to be given the token and somehow have it applied (how that happens is dependent on the service, it may or may not require a restart/reload/nothing).

Reply:
I added another note at the beginning of the section. There was some language about privileges in the instructions, but having a general note a the top seems good too. @mkeeler let me know if the new info at the beginning of the section needs additional updates.